search/templates/search/search.html

```diff
@@ -10,6 +10,56 @@
   });
 });
 </script>
+
+<style>
+.meter {
+    width: 4rem;
+    height: 0.7rem;
+    position: relative;
+    background: #555;
+    -moz-border-radius: 25px;
+    -webkit-border-radius: 25px;
+    border-radius: 25px;
+    box-shadow: inset 0 -1px 1px rgba(255,255,255,0.3);
+}
+
+div.meter {
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.meter > span {
+  display: block;
+  height: 100%;
+  border-top-right-radius: 8px;
+  border-bottom-right-radius: 8px;
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+  background-color: rgb(43,194,83);
+  background-image: linear-gradient(
+    center bottom,
+    rgb(43,194,83) 37%,
+    rgb(84,240,84) 69%
+  );
+  box-shadow:
+    inset 0 2px 9px  rgba(255,255,255,0.3),
+    inset 0 -2px 6px rgba(0,0,0,0.4);
+  position: relative;
+  overflow: hidden;
+}
+
+.horizontal-list {
+    display: flex;
+    list-style-type: none;
+}
+.horizontal-list li {
+    margin-left: 1rem;
+}
+#stats-list {
+    margin: 0;
+    padding: 0;
+}
+</style>
```

Comment:
Do these styles need to be inline? If they can't be put in the main styles.css sheet, could they be put in their own stylesheet that could be imported to this page?

Reply:
I didn't want to put in styles.css because it's globally imported across pages and there's a risk of breaking the style of other pages without noticing it (e.g css class name conflicts)

And putting it in their own stylesheet sounds good to me and I love to do it. I think the more our frontend code get complicated, the more important it is. However it looks pretty simple yet from my perspective. That's why I just put it there.